server/src/main/java/org/opensearch/action/search/UpdatePitContextResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.search;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.transport.TransportResponse;
+
+import java.io.IOException;
+
+/**
+ * Update PIT context response with creation time, keep alive etc.
+ */
+public class UpdatePitContextResponse extends TransportResponse {
```

Comment:
Could we please follow the same naming convention: either use `XxxPit` or `XxxPIT`?